Control/lib/typedefs/TypeMap.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef {Object.<string, TypeMapEntry>} TypeMap
+ * 
+ * Object which is a map of types.
+ * Keys are taken from existing configuration.
+ * Values are describing what is expected type of value held there.
+ * 
+ * For example for the given configuration:
+ * ```
+ *  {
```

Comment:
It would be better if you use a qc alike configuration object rather than invented one